instrumentation/kafka-clients-node-metrics-1.0.0/src/main/java/com/nr/instrumentation/kafka/CachedKafkaMetrics.java

```diff
@@ -12,18 +12,6 @@ static CachedKafkaMetric newCachedKafkaMetric(final KafkaMetric metric) {
             return new CachedKafkaVersion(metric);
         }
 
-        Measurable measurable = null;
```

Comment:
Since there was a split in the modules, the versions supported by this do not support CumulativeSum.